<launch>
  
  <!-- <param name="robot_description" textfile="$(find
  aass_cyberglove)/model/cyberglove.xml"/>-->

  <node pkg="aass_cyberglove" name="aass_cyberglove" type="aass_cyberglove" output="screen">
    <param name="cyberglove_prefix" type="string" value="/aass_cyberglove" />
    <param name="publish_frequency" type="double" value="30.0" />
    <param name="path_to_glove" type="string" value="/dev/ttyUSB0" />
  </node>

</launch>
